### PR TITLE
game-theoryst:0.1.0

### PR DESCRIPTION
**Pull request by connortwiegand**
I am submitting
- [X] a new package
- [ ] an update for a package

Description: Typeset beautiful Normal/Strategic Form Games with Typst. Supports underlining, showing mixed strategy parameters, and eliminating strategies via the `pinit` package.

I have read and followed the submission guidelines and, in particular, I
- [X] selected a name that isn't the most obvious or canonical name for what the package does
- [X] added a `typst.toml` file with all required keys
- [X] added a `README.md` with documentation for my package
- [X] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [X] tested my package locally on my system and it worked
- [X] `exclude`d PDFs or README images, if any, but not the LICENSE

### CODE REVIEW

File: packages/preview/game-theoryst/0.1.0/src/utils/table_utils.typ [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#let blank_cells(cspn: 3) = {
+  cell(colspan: cspn, [])
+}
+
+// "Stroke-if": utility to stroke only the payoff matrix, not the player or strategy cells
+#let strf(x, y, tstart: 3) = {
+  if-else(x < tstart or y < tstart, none, 1pt)
+}
+
+"Align-if": Utility to set alignment for players and strategies
```
Comment (elegaanz) [inline packages/preview/game-theoryst/0.1.0/src/utils/table_utils.typ:15]: I think a `//` is missing here.

```suggestion
// "Align-if": Utility to set alignment for players and strategies
```

File: packages/preview/game-theoryst/0.1.0/src/utils/table_utils.typ [hunk 2]
```diff
@@ -0,0 +1,36 @@
+#import table: cell
+#import "programming_utils.typ": if-else
+
+
+// Utility to insert blank cells
+#let blank_cells(cspn: 3) = {
```
Comment (elegaanz) [inline packages/preview/game-theoryst/0.1.0/src/utils/table_utils.typ:6]: Typst packages generally use kebab-case names. Can you please rename this function to `blank-cells`? Kebab case is only a convention, so if you want to keep it as is, it is also possible, just tell me what you prefer.

Comment (connortwiegand) [reply packages/preview/game-theoryst/0.1.0/src/utils/table_utils.typ]: Updated to reflect both changes. Thank you!

**Comment by elegaanz** [comment]
Thanks :) LGTM

**Pull request merged**